cmake/std/PullRequestLinuxDriverTest.py

```diff
@@ -435,14 +443,14 @@ def createPackageEnables(arguments):
                                    'packageEnables.cmake'])
         else:
             with open('packageEnables.cmake',  'w') as f_out:
-                f_out.write('''
-MACRO(PR_ENABLE_BOOL  VAR_NAME  VAR_VAL)
-  MESSAGE("-- Setting ${VAR_NAME} = ${VAR_VAL}")
-  SET(${VAR_NAME} ${VAR_VAL} CACHE BOOL "Set in $CMAKE_PACKAGE_ENABLES_OUT")
-ENDMACRO()
-
-PR_ENABLE_BOOL(Trilinos_ENABLE_''' + enable_map[arguments.job_base_name] + ''' ON)
-''')
+                f_out.write(textwrap.dedent("""\
+                    MACRO(PR_ENABLE_BOOL  VAR_NAME  VAR_VAL)
+                        MESSAGE("-- Setting ${VAR_NAME} = ${VAR_VAL}")
+                        SET(${VAR_NAME} ${VAR_VAL} CACHE BOOL "Set in $CMAKE_PACKAGE_ENABLES_OUT")
+                    ENDMACRO()
+
+                    PR_ENABLE_BOOL(Trilinos_ENABLE_''' + enable_map[arguments.job_base_name] + ''' ON)
```

Comment:
These triple quotes must also change to """ - I  looked in the testing results and found that they are being written directly as 

PR_ENABLE_BOOL(Trilinos_ENABLE_''' + enable_map[arguments.job_base_name] + ''' ON)

Reply:
ok. I'll get that done this morning.